### PR TITLE
checks if bonded is signed in transactions history list 

### LINKED ISSUE #1488: Aggregate bonded - Cosigner have to open tx details to check if it has to be signed
Steps:

1. Announce a few aggregates bonded from multisig account with minimum approval 3 (one cosigner)
2. Go to the second cosigner and sign one of the partial transactions.
3. Logout and login because of bug https://github.com/nemgrouplimited/symbol-desktop-wallet/issues/1487
4. Go to the transactions list. Now you can see a few partial transactions. One of them is already signed by this cosigner but there is no way to tell which one without opening details.

Expected result: It should be already visible on the list that the account has to sign partial.

### PR DESCRIPTION
**Pull request by bassemmagdy**
fixes #1488

### CODE REVIEW

File: src/components/TransactionList/TransactionRow/TransactionRowTs.ts [hunk 1]
```diff
@@ -44,6 +57,9 @@ import { DateTimeFormatter } from '@js-joda/core';
         explorerBaseUrl: 'app/explorerUrl',
         networkConfiguration: 'network/networkConfiguration',
         currentProfile: 'profile/currentProfile',
+        currentAccount: 'account/currentAccount',
+        currentAccountMultisigInfo: 'account/currentAccountMultisigInfo',
+        generationHash: 'network/generationHash',
```
Comment (AnthonyLaw) [inline src/components/TransactionList/TransactionRow/TransactionRowTs.ts:62]: I did find `generationHash` use on this code, do we need it?

File: src/components/TransactionList/TransactionRow/TransactionRowTs.ts [hunk 2]
```diff
@@ -85,6 +101,32 @@ export class TransactionRowTs extends Vue {
...
+     * Currently active account
+     * @see {Store.Account}
+     * @var {AccountModel}
+     */
+    private currentAccount: AccountModel;
+    /**
+     * Current network's generation hash
+     * @var {string}
+     */
+    public generationHash: string;
```
Comment (AnthonyLaw) [inline src/components/TransactionList/TransactionRow/TransactionRowTs.ts:114]: I did find `generationHash` use on this code, do we need it?

File: src/language/zh-CN.json [hunk 3]
```diff
@@ -1117,5 +1117,6 @@
     "optin_postlaunch_tx_completed_amount": "您已收到的金额：",
     "optin_postlaunch_tx_nis_address": "选择加入NEM NIS1账户",
     "show_details": "显示详细信息",
-    "hide_details": "隐藏详细信息"
+    "hide_details": "隐藏详细信息",
+    "transaction_signed": "签约成功"
```
Comment (AnthonyLaw) [inline src/language/zh-CN.json:1121]:
```suggestion
    "transaction_signed": "已签"
```

**Comment by AnthonyLaw** [comment]
Added diagram.

Screenshot 2021-06-09 at 3 40 16 PM

File: src/components/TransactionList/TransactionRow/TransactionRowTs.ts [hunk 4]
```diff
@@ -194,13 +236,103 @@ export class TransactionRowTs extends Vue {
...
+        // Multisig account can not sign
+
+        const currentPubAccount = AccountModel.getObjects(this.currentAccount).publicAccount;
+        if (
+            this.transaction instanceof AggregateTransaction &&
+            this.transaction.type === TransactionType.AGGREGATE_BONDED &&
+            !this.transaction.signedByAccount(currentPubAccount)
+        ) {
+            this.transactionSigningFlag = true;
+            setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (bassemmagdy) [reply src/components/TransactionList/TransactionRow/TransactionRowTs.ts:249]: fetching transaction details is an async action, so had to put some delay to make sure it's fetched already

File: src/components/TransactionList/TransactionRow/TransactionRowTs.ts (hunk 1 above)
Comment (rg911) [reply src/components/TransactionList/TransactionRow/TransactionRowTs.ts:62]: @AnthonyLaw  it is from the `network/generationHash` network store.

File: src/components/TransactionList/TransactionRow/TransactionRowTs.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (rg911) [reply src/components/TransactionList/TransactionRow/TransactionRowTs.ts:249]: hmm. how do we know 1 sec is enough? is it possible to use waiter?

File: src/components/TransactionList/TransactionRow/TransactionRowTs.ts [hunk 5]
```diff
@@ -194,13 +236,103 @@ export class TransactionRowTs extends Vue {
...
+        }
+    }
 
     /**
      * Returns the transaction height or number of confirmations
      */
     public getHeight(): string {
         const transactionStatus = TransactionView.getTransactionStatus(this.transaction);
-        if (transactionStatus == TransactionStatus.confirmed) {
+        if (transactionStatus === 'confirmed') {
```
Comment (rg911) [inline src/components/TransactionList/TransactionRow/TransactionRowTs.ts:335]: why not using the enum?

File: src/components/TransactionList/TransactionRow/TransactionRowTs.ts (hunk 1 above)
Comment (AnthonyLaw) [reply src/components/TransactionList/TransactionRow/TransactionRowTs.ts:62]: yeah. I mean `generationHash` is not using anywhere on `TransactionRowTs.ts`

**Review by AnthonyLaw — APPROVED** [review]
tested, it's working.
it look good for me.

**Pull request merged**